doc/rst/technotes/gpu.rst

```diff
@@ -113,39 +113,39 @@ Setup
 Requirements
 ~~~~~~~~~~~~
 
-* ``LLVM`` must be used as Chapel's backend compiler (i.e.
-  ``CHPL_LLVM`` must be set to ``system`` or ``bundled``). For more information
-  about these settings see :ref:`Optional Settings <readme-chplenv>`.
+First, please make sure you are using Chapel's `preferred configuration
+<../usingchapel/QUICKSTART.html#using-chapel-in-its-preferred-configuration>`_
+as the starting point. Specifically, the "quickstart" configuration can not be
+used with GPU support.
 
-  * If using a ``system`` LLVM:
+The following are further requirements for GPU support:
 
-    * it must have been built with support for the relevant target of GPU you
-      wish to generate code for (e.g.  NVPTX to target NVIDIA GPUs and AMDGPU to
-      target AMD GPUs).
+* For targeting NVIDIA or AMD GPUs, ``LLVM`` must be used as Chapel's backend
+  compiler (i.e.  ``CHPL_LLVM`` must be set to ``system`` or ``bundled``).
 
-    * we expect it to be the same version as the bundled version (currently 17).
-      Older versions may work; however, we only make efforts to test GPU support
-      with this version.
+* Specifically for targeting NVIDIA GPUs:
 
-* Either the CUDA toolkit (for NVIDIA), or ROCm (for AMD) must be installed
-  (unless using `CPU-as-Device mode`_)
+  * CUDA toolkit version 11.x or 12.x must be installed.
```

Comment:
No longer true that CUDA 12 needs bundled?

Reply:
Yes, but I may have over-deleted stuff. When using CUDA 12, you have to use LLVM 16 or 17. Doesn't matter system vs bundled. That's a CUDA/LLVM compat issue, and not on us. I'll add a subbullet under this one for this.